user/tutorials/User Guide for Translation Tool/User Guide for the Translation Tool.md

```diff
@@ -0,0 +1,48 @@
+
+
+| title                               | category | description                             |
+| ----------------------------------- | -------- | --------------------------------------- |
+| User Guide for the Translation Tool | tutorial | Explains how to use the Tranlation Tool |
+
+# Instructioins
+
+1. Log in to an account with the translation privilege on the dev website.
+
+   ![Step 1](https://github.com/ohmewho/appbuilder_docs/blob/master/user/tutorials/User%20Guide%20for%20Translation%20Tool/images/Step%201.png)
```

Comment:
For the links please use a relative link `(images/Step%201.png)`
This references your GitHub fork rather than the images that will be in the main repo.
[More details in absolute vs relative paths](http://www.geeksengine.com/article/absolute-relative-path.html)

Reply:
Thanks for fixing this one, please change all the other links in the file as well.